### DISCUSSION TITLE
When your done using the sh1mmer gui and reboot does it automaticly un-enroll

### DISCUSSION THREAD

**Discussion by Yashvovxb** [3kh0/ext-remover#504] [Help JOIN TITANIUM NETWORK TO SHARE PLEASE] [answered]
Wanted to know because my admins would get mad.

**Comment by fnfgo** [comment, marked as answer]
no

**Comment by amelia-bartlett** [comment]
It does not auto unenroll, you have to manually take these actions:
1)Load the sh1mmer shim [using the normal RMA shim boot procedure](https://chromium.googlesource.com/chromiumos/platform/factory/+/HEAD/setup/RMA_SHIM.md#Boot-from-RMA-shim-clamshells-convertibles).
2)Get past the (false) warning screen (the GPL3 does not prohibit charging for distribution, only _making a profit when distributing source code on request after distributing a binary build_, and the FSF - the authors of the GPL3 text explicitly encourage charging for distribution of software under the license)
3)Select Utilities
4)In any order: select "Wipe GBB flags" and "Enable devmode" (enable devmode also loads the disable enrollment script automatically, at least on the version of the source I built from, you may wish to run the unenroll utility as well to be sure)
5)After all of the above utilities have been run, press backspace to go back to the main menu
6)Select "Reboot"
7)Follow instructions to return to verified boot mode (some chromeos recovery firmware builds call this "secure mode", if yours is **NOT** like this, hit spacebar)
8)Sign in with a **non-managed google account**.  This can be a pre-existing home account, or one you create on the spot - as long as it is not your organisation (i.e. school/workplace provided) account it is fine
9)You should enable devmode using the standard ChromeOS procedure for it _now_ if you want it - enabling it later will still work, but it will wipe the local disk.
10)Enjoy your unmanaged chromebook!
(If you need to sign into a managed account then you can, just don't do it first!  If you need to do it to take e.g. a google forms locked mode quiz or a kiosk mode test, follow [these instructions](https://sh1mmer.me/kiosks.txt) - note these instructions _require_ you to be in devmode, also note if you have no forward arrow on your keyboard, use refresh instead)
